neural_solution/docs/source/README.md

```diff
@@ -126,3 +131,32 @@ There are several logs under workspace:
 
 ```
 
+## Start resource management
+Neural Solution supports cluster management for service maintainers, providing several command-line tools for efficient resource management. 
+
+### Node States
+
+Each node in the cluster can have three different states:
+
+- Alive: Represents a node that is functioning properly and available to handle requests.
+- Join: Indicates that a node is in the process of being added to the cluster but has not fully joined yet.
+- Remove: Indicates that a node is scheduled to be removed from the cluster.
+
+Below are some commonly used commands and their usage:
+
+### Query cluster
+This command is used to query the current status of the cluster. No additional parameters are required, simply enter the following command:
+```shell
+neural_solution cluster --query
+```
+### Add node
+This command is used to add nodes to the cluster. You can either specify a host file or provide a list of nodes separated by ";". For example:
+```shell
+neural_solution cluster --join "host1 2 20; host2 4 20"
```

Comment:
Could you provide more details regarding the node format?  
e.g., `hostname number_of_sockets cores_per_socket`